changed 2019->2020

nlok5923 opened the pull request:
As requested replaced all 2019 with 2020

fixes  : #18

aashutoshrathi commented on about.md:
```diff
@@ -33,7 +33,7 @@
 Sabar Hostel premises
 # <h3>Time:</h3>
-Date: 5th October 2019 (Saturday) 10:00 AM to 6th October 2019 (Sunday) 10:00 AM.
+Date: 5th October 2029 (Saturday) 10:00 AM to 6th October 2020 (Sunday) 10:00 AM.
```
Weekdays are same?

nlok5923 replied:
changed weekdays in the latest commit

aashutoshrathi commented on app/client/views/about/about.html:
```diff
@@ -54,7 +54,7 @@ <h3>Venue</h3>
     </div>
     <br>
-    <div style="text-align: center;"><b> Date: 5th October 2019 (Saturday) 10:00 AM to 6th October 2019 (Sunday) 10:00
+    <div style="text-align: center;"><b> Date: 5th October 2020 (Saturday) 10:00 AM to 6th October 2020 (Sunday) 10:00
```
Same doubt here.

nlok5923 replied:
changed weekdays in the latest commit

aashutoshrathi commented on app/client/views/admin/user/user.html:
```diff
@@ -110,10 +110,10 @@
-          <option value="2020">2020</option>
-          <option value="2021">2021</option>
-          <option value="2022">2022</option>
+          <option value="2019">2020</option>
```
Update the values here

nlok5923 replied:
values updated

aashutoshrathi commented on app/client/views/application/application.html:
```diff
@@ -51,10 +51,10 @@
-            <option value="2020">2020</option>
-            <option value="2021">2021</option>
-            <option value="2022">2022</option>
+            <option value="2019">2020</option>
```
Values are not updated

nlok5923 replied:
values updated

aashutoshrathi commented on app/server/templates/email-link-action/text.hbs:
```diff
@@ -1,4 +1,4 @@
+Hi! Thanks for signing up for HackIIITV 2020. To verify your email, follow this link: {{verifyUrl}}
 
 Thanks,
 HackIIITV Team
```
Newline preferred.

nlok5923 replied:
Newline inserted in the latest commit.
changes done as requested please review @aashutoshrathi sir.

nlok5923 commented:
@iiitv/project-maintainers sir please review this pull request.

aashutoshrathi reviewed (changes requested):
Can you check why diff shows complete files instead of just `change`

nlok5923 commented:
Sir, because previously in my vscode I had modified setting to format on save because of that after every edit the code gets formatted. To resolve the same issue I had removed those setting from my vscode and after that made changes to the code.

nlok5923 commented:
opened a new #24 pull request for the same issue please review it.
